### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by leventt**
fixed my last name

### CODE REVIEW

**Comment by Lee-JaeWon** [comment]
@leventt

This repository organizes results that are automatically crawled from arXiv. It appears that your last name is sometimes incorrectly parsed during the crawling process on arXiv, and I may not be able to manually correct it each time. I will accept this PR, but please note that future updates might not reflect the correction.

Thank you for your interest.

**Comment by leventt** [comment]
:) thanks for your explanation! cheers

**Comment by Lee-JaeWon** [comment]
@leventt 
I've been reading your paper more closely, and I have a question that you would be happy to answer.

I have two questions,

The first question is more of a personal curiosity: What exactly is Netflix Eyeline Studio? Is it an organization dedicated to creating CG work for Netflix content, or is it purely for research and development purposes? I apologize if this question seems basic—I was just genuinely curious.

The second question pertains to your paper.
Based on my understanding, it seems that you fine-tuned a diffusion model for image-to-image translation using rendered images and images illuminated with specific lighting conditions. However, the results in your paper’s visualization (video) are so natural that I couldn’t help but wonder if these diffusion results were reapplied to Gaussian points. Did I misunderstand this aspect? Thank you for your fantastic paper, and I would greatly appreciate your response.

**Comment by leventt** [comment]
Hi Lee JaeWon,

First of all thank you for your interest, time and kind words.

Netflix Eyeline Studios is a production innovation hub for Netflix. Netflix
relatively recently acquired Eyeline Studios along with the better known
parent studios called Scanline VFX. We try to balance research with
production. You can find more about it here:
https://scanlinevfx.com/
https://www.eyelinestudios.com/

Our method we present in the paper consists roughly of two independent
parts. First we reconstruct a performance with our novel scaleable approach
using segments of gaussian primitives. Each segment in time would also have
an associated MLP. This MLP represents a deformation field for a segment
typically of tens of frames. These segments would have some overlapping
frames. Typically we linearly interpolate the splatting results over 5
overlapping frames so the mismatches between segments aren’t perceived.
Separately we fine tune a latent diffusion model with existing weights
after repeating the weights of the first convolution layer of the denoising
UNet. This allows us to concatenate 4 more channels of noise to the latent
image. This idea is borrowed from the popular marigold framework, which is
typically used for predicting depth images. We instead use it for image
translation. We also concatenate a vector of repeated SH encoded light
direction with a magnitude representing the area of the light in reverse
proportion. So a magnitude of 1 is the smallest area our stage supports and
0 denotes light coming from all directions while the energy being the same.
We have ground truth data from our custom stage. We use this in conjunction
with measured light and camera positions, because we condition the light
encoding always in camera space. This helps us apply our relighting model
without needing a scene where the camera is known or any geometry
reconstruction or guidance. We then use these two parts together. GS part
of the framework allows us to change views while the image translation part
with the custom condition used in place of the text embedding allows us to
change light direction. We infer high dynamic range linear color images
that match ground truth in a plausible way. One of the insights that made
this possible was the use of multi-resolution pyramid noise during fine
tuning. This allows us to reproduce color and contrast better.

I believe this may be a good enough summary I can write on my phone but
please feel free to ask me to elaborate or clarify.

Again thank you for your interest and time,
Ahmet Levent Taşel

**Comment by Lee-JaeWon** [comment]
@leventt 
In the paper, the method uses SH (Spherical Harmonics) encoding for lighting directions.

I am struggling to understand how this encoding works in the context of inference.
Specifically, during inference, if we only provide the view or light direction as input, does the model also generate the colors and lighting effects automatically?

Could you explain in detail how the SH encoding influences the generation of lighting and color information during inference?

**Comment by leventt** [comment]
From camera and light positions and measured scene center we derive a
direction in relation to each of the ground truth views. Training this way
removes the need to provide both camera and light positions as the
condition includes both of them in relation to each other, this way.
Positive X would point screen right, Y would point up and Z out of the
screen towards the viewer. During inference, we provide this condition in
addition to the flat-lit images we render from GS. Diffusion network
effectively memorizes the shape and appearance of the subject specific to
the identity of the performer. We show that this improves accuracy over
generalizing with the limited number of performers.

SH encoding provides a way for the model to recall and interpolate to novel
views and light conditions in tandem with the flat-lit input that gets
concatenated with the latent noise.
It is a model that intentionally overfits an identity and we show how this
overfitting or generalization may compare in terms of accuracy (Fig. 8)

Offloading the appearance modeling to be done implicitly using a data
driven approach such as ours, seems to perform better than explicit
modeling of appearance, which is the de facto standard for CG productions.
However this method obviously requires the stage and the data from it which
is not as wide spread.
